src/akkd/Accordion/AccordionPanel.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames'
+import React from 'react'
+import styles from './Accordion.module.css'
+import {AccordionContent} from './AccordionContent'
+import {AccordionTitle} from './AccordionTitle'
+
+interface Props {
+	panel: any,

```

Comment:
if you `export` the Panel interface from the other file, should be able to import it for this type